charts/zeebe-benchmark/test/golden/c8-core-configmap.golden.yaml

```diff
@@ -117,8 +116,20 @@ data:
           enabled: true
 
       security:
+        authentication:
```

Comment:
:question: There are several identity related changes do we know how it affects our benchmarks?